sk8prk/assets/block-extends/_image.scss

```diff
@@ -0,0 +1,24 @@
+.wp-block-image {
+    &.is-style-duotone {
```

Comment:
All of the duotone effects are created with CSS and blend-modes, and applied with block style variations.

Reply:
We should double check how these render on IE (even the later versions). I remember some issues with duotone images on Twenty Nineteen.